.mcignore

```diff
@@ -14,6 +14,9 @@ npm-debug.log
 /logs/
 /node_modules/
 
+# ignore README since it's GitHub specific
+/README.md
```

Comment:
We don't really need to remove it but I guess it's fine if we do. But if we do keep it in mozilla-central, we could note that the doc is GitHub focused and to see https://firefox-source-docs.mozilla.org/browser/components/newtab/docs/ (assuming we name the url path "docs" with `SPHINX_TREES['docs'] = 'docs'` where the left "docs" is the url path and the right "docs" is our directory structure)